src/Verifier/SAW/Simulator/RME.hs

```diff
@@ -214,6 +214,9 @@ prims =
   , Prims.bpIntMax = undefined--pure2 max
```

Comment:
It's sort of unrelated to your change, but we now have the opportunity to treat this `undefined` in a better way.

Reply:
I am doing that in SBV, but I missed it here. Thank you for spotting it!